src/lambda_codebase/account_processing/process_account_files.py

```diff
@@ -33,14 +35,15 @@ def get_details_from_event(event: dict):
     object_key = s3_details.get("object", {}).get("key")
     return {
         "bucket_name": bucket_name,
-        "object_key": object_key,
+        "key": object_key,
     }
 
 
 def get_file_from_s3(s3_object: dict, s3_resource: boto3.resource):
     try:
+        LOGGER.debug(json.dumps(s3_object, indent=2) if LOGGER.isEnabledFor(logging.DEBUG) else "--data-hidden--")
```

Comment:
Could you add some context on where this debug is located, like so?

```suggestion
        LOGGER.debug(
            "Reading YAML from S3: %s",
            json.dumps(s3_object, indent=2) if LOGGER.isEnabledFor(logging.DEBUG) else "--data-hidden--"
        )
```